spec/models/searchworks_item_spec.rb

```diff
@@ -0,0 +1,58 @@
+require 'rails_helper'
+
+describe SearchworksItem do
+  let(:item_id) { SearchworksItem.new }
```

Comment:
So instead of `item` I am using `subject` now...
let(:subject) { SearchworksItem.new('123') }

This works:
`expect(subject.title?).to be true`
But not this:
`expect(subject).to have_title?`